composer.json

```diff
@@ -13,7 +13,7 @@
         "php": "^7.4",
         "ext-json": "*",
         "ext-mbstring": "*",
-        "jdorn/sql-formatter": "dev-master",
+        "jdorn/sql-formatter": "^1.2",
```

Comment:
Takze problem bol v `jdorn/sql-formatter` a revertlo sa to z `2015` na `2013`? 
Myslim si, ze ked sme to implementovali, tak najlos alebo odin nam k tomu davali nejaky komentar, a tam to odporucali dat na `dev-master` (alebo to bolo tak v nejakom inom repe a skopirovali sme to). Je to tak aj v `synapse-transformation`.

Bolo by dobre overit, ze tam nie su potrebne nejake commity, ktore su teraz iba v master vetve.

Reply:
bohužel ten `dev-master` není úplně v pohodě v tomhle případě protože format sqlka udělal z `"account_id"::INT` chybnou syntax `"account_id" : : INT`

to že nám doporučovali použít `dev-master` si nepamatuji a že to je i v synapse bude tím že jsme to zkopírovali odsud :)

Reply:
imho to dělá tento commit - https://github.com/jdorn/sql-formatter/commit/ffecdad6ca3f6235f941e960bc9d290a20054586

Reply:
no hovorili sme urcite o tom, ze najlepsie by bolo ten package forknut a mat stabilnu verziu pod keboola .... 
ok, ak tvrdis ze to pojde bez problemov, davam approve ;)

Reply:
aha :D https://github.com/keboola/sql-formatter

Reply:
ok, ja o tom dokopy nic neviem, najlepsie by sa bolo opytat Najlosa - ze co by bolo dlhodobe riesenie.

Reply:
ked ti to s tym forkom pojde, tak asi nie je co riesit.

Reply:
on ten `dev-master` je totiž z našeho upravenýho repa:
```
"repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/keboola/sql-formatter"
        }
    ],
```

připravil jsem PR v našem forku
https://github.com/keboola/sql-formatter/pull/2